beta/nncf/tensorflow/quantization/algorithm.py

```diff
@@ -123,10 +140,11 @@ def get_transformation_layout(self, model):
 
             weight_attr_name = QUANTIZATION_LAYERS[node['type']][WEIGHT_ATTR_NAME]
             op_name = self._get_quantizer_operation_name(node_name, weight_attr_name)
+            self._op_names.append(op_name)
 
             operation = self._create_quantizer(op_name, TFQuantizerSpec.from_config(qconfig,
-                                                                           narrow_range=True,
-                                                                           half_range=False))
+                                                                           narrow_range=False,
```

Comment:
I guess we should have the following logic:

```
narrow_range = (half_range==False):   
```

Reply:
Done